jobs/tsconfig.json

```diff
@@ -3,7 +3,8 @@
 	"compilerOptions": {
 		"tsBuildInfoFile": "node_modules/.cache/tsbuildinfo.json",
 		"lib": ["ESNext"],
-		"types": ["node"]
+		"types": ["node"],
+		"moduleResolution": "bundler"
```

Comment:
This is needed for type-checking to pass, which I _think_ is correct as we are just straight running this with `tsx`, not building this package as an output. However, this is mostly figured out by trial and error, not by some very deep understanding of tsconfig best practices